examples/one_dot_one/src/generatedNoCheck/Ontology.ts

```diff
@@ -15,7 +15,7 @@ export const Ontology: {
   metadata: {
     ontologyRid: 'ridHere';
     ontologyApiName: 'OntologyApiName';
-    userAgent: 'foundry-typescript-osdk/0.0.1';
+    userAgent: '@osdk/cli/0.1.1';
```

Comment:
^ this is one of a few places that demonstrates that the @osdk/cli useragent injection is working with the full version string.